### PR TITLE
slight bit of file optimisation

### PR DESCRIPTION
**Pull request by rhysdh540**
my estimate is about 40% reduced size, might be a little larger or smaller

also added .DS_Store to .gitignore (common mac L)

### CODE REVIEW

File: assets/create/textures/gui/overlay.png
Comment (CodeF53) [inline assets/create/textures/gui/overlay.png:1]: Shouldn't have rounded corners, use wrap mode

File: assets/create/textures/gui/filters.png
Comment (CodeF53) [inline assets/create/textures/gui/filters.png:1]: See prior comment on slot grids

File: assets/create/textures/gui/value_settings.png
Comment (CodeF53) [inline assets/create/textures/gui/value_settings.png:1]: Most of this texture should use relayer for the corners, with an exception for the curly bracket esque shapes.

**Review by CodeF53 — CHANGES_REQUESTED** [review]
Gui textures are a lot harder to get right and need manual work.

After addressing review please look at how they look in-game as well.

**Comment by rhysdh540** [comment]
Yeah sorry I forgot I put those in, they weren't the main intention of this, I just added them before I realized this wasn't 0.5.1 compatible. Since I'm "working" on that (I'm really not but whatever) right now and making these changes would be useless since I would end up redoing them later, I'll just remove the guis.

**Comment by rhysdh540** [comment]
Also apparently I deleted all the create textures, oops, I'll put those back for the time being.

**Comment by CodeF53** [comment]
Also if we are adding GUI textures then we need to support EMI and JEI or else the mod will just look silly. As those are always installed alongside Create.
But that can go in a seperate pull request

**Comment by rhysdh540** [comment]
image :D

**Pull request merged**